benchmark/atoi-catch2-tests.cpp

```diff
@@ -0,0 +1,18 @@
+#include "catch2/catch.hpp"
+
+#include "zoo/swar/SWAR.h"
+
+uint64_t calculateBase10(zoo::swar::SWAR<8, __uint128_t>) noexcept;
+
+TEST_CASE("Calculate Base10", "[pure-test][swar][atoi]") {
```

Comment:
Perhaps more tests? something unaligned?

Reply:
Yes, the `atol` that straddles two blocks was unit-tested